src/commands/status/docker.js

```diff
@@ -0,0 +1,54 @@
+const {table} = require('table');
+
+const BaseCommand = require('../../oclif/command/BaseCommand');
+
+const MuteOneLineError = require('../../oclif/errors/MuteOneLineError');
+
+const PRESETS = require('../../presets');
+
+class DockerStatusCommand extends BaseCommand {
+  /**
+   * @param {Object} args
+   * @param {Object} flags
+   * @param {DockerCompose} dockerCompose
+   * @return {Promise<void>}
+   */
+  async runWithDependencies(
+    {
+      preset,
+    },
+    flags,
+    dockerCompose,
+  ) {
+    const data = [];
+    let output;
+
+    const tableConfig = {
+      //singleLine: true,
+      drawHorizontalLine: (index, size) => {
+        return index === 0 || index === 1 || index === size;
+      }
+    };
+
+    data.push(['Container', 'ID', 'Status']);
+    data.push(...(await dockerCompose.listContainerStatus(preset)));
```

Comment:
Shouldn't we ignore `drive_mongodb_replica_init`? It's not supposed to running and could confuse people? Don't we want to add human-readable service names?

Reply:
I would also use colors for Status column.

Reply:
I currently get the service names by inspecting each container returned by the `docker ps -q` command. I can't see any friendly service names in that output, the best I could find were the friendly names we define in the `docker-compose.*.yml` files. Are these friendly enough? Or are you suggesting I convert these names into hard coded names in the CLI?

I'm not sure if I should specifically ignore `drive_mongodb_replica_init` in the command output, maybe we could show when it was last run? What does it actually do, does it only run once or regularly? I think it's better to include it and leave a note on docs.dash.org that it is normal for this service not to be running.

Should I use chalk for colors?

Reply:
Maybe we should just hard code list of services with human-readable names and inspect this list? Or we could have a mapping?

Reply:
the table lib doesn't prove colors for cells? we could use chalk as well.

Reply:
`drive_mongodb_replica_init` starts only to setup replica set and then exit. So it's normal behavior when it exit. Maybe we should only hide it when it exit?

Reply:
OK, this is done as well now. I also added human-readable service names, but I don't really think this adds anything useful to the output - it's basically the same information. I have filtered out `drive_mongodb_replica_init` from the results completely, debugging this service can just directly use `docker logs` for advanced users I think?

Reply:
Looks good. I've refactored a bit your code and sent you diff in Slack

Reply:
Thanks, this is a huge help to learn better coding practices! What you have done is basically separating the concerns in a more logical way, right? In general, how do you reason about which logic can be extracted into a separate function? Not sure if this is something you can learn or if it just comes from experience.

My only question is about the logic of having the `getContainersList()` function return an array of strings when both `inspectService()` and `isServiceRunning()` will ignore everything except the first string in the array. The previous design of `inspectService()` allowed it to process one or more services at a time. What is the advantage of moving the loop logic into the status command? Should we simplify `getContainersList()` to only return a single container ID instead of an array? Because now it is only ever called together with a `serviceName`, so it only ever returns a single result.

Reply:
> What you have done is basically separating the concerns in a more logical way, right? In general, how do you reason about which logic can be extracted into a separate function?

Usually, if you need to reuse this part of code somewhere else it make sense to extract it to a separate function in order to do not duplicate code.

> My only question is about the logic of having the getContainersList() function return an array of strings when both inspectService() and isServiceRunning() will ignore everything except the first string in the array...

It's possible for a service to have several containers, in case of failure or using the `run` command. I guess we should make `inspectService()` to respond with an array of inspections for the service and handle it in the docker command class.

Reply:
Sorry for the long delay, I was overloaded with other work. I don't really understand what you mean here, I was thinking more about how the functions pass data around internally rather than any external services with multiple containers. I think it works now, so it's not really important to modify this behaviour until we need it?

Reply:
I mean, practically, it possible that a docker-compose service might have several containers in some cases. For example, in ps output, the first one could be exited due to an error, and the second one created after a service restart. So on your list, you will show just that service is exited but it’s not really true.
We don’t need to handle it right now. Let’s keep it simple :)